packages/renderer/src/lib/kube/details/KubePortsList.spec.ts

```diff
@@ -22,8 +22,10 @@ import { render } from '@testing-library/svelte';
 import { readable } from 'svelte/store';
 import { beforeEach, expect, test, vi } from 'vitest';
 
-import KubeContainerPorts from '/@/lib/kube/details/KubeContainerPorts.svelte';
 import * as kubeContextStore from '/@/stores/kubernetes-contexts-state';
+import { WorkloadKind } from '/@api/kubernetes-port-forward-model';
+
+import KubePorts from './KubePortsList.svelte';
```

Comment:
Same as above

Reply:
Yes, nice catch, I renamed it twice, sadly the IDE is not smart enough to rename the import :( Updated !